app/src/main/java/org/simple/clinic/util/UuidTypeConverters.kt

```diff
@@ -18,6 +18,32 @@ class UuidRoomTypeConverter {
   }
 }
 
+class UuidListRoomTypeConverter {
+
+  private val uuidConverter = UuidRoomTypeConverter()
+
+  @TypeConverter
+  fun toUuids(value: String?): List<UUID>? {
+    if (value == null) {
+      return null
+    }
+
+    return value.split(delimiters = *arrayOf(","))
+        .map { uuidConverter.toUuid(it)!! }
+        .toList()
+  }
+
+  @TypeConverter
+  fun fromUuids(uuids: List<UUID>?): String? {
+    if (uuids == null) {
```

Comment:
Since the return type is a nullable string, we can make this a single expression function:

```kotlin
fun fromUuids(uuids: List<UUID>?) = uuids?.joinToString(separator = ",", transform = { uuid -> uuidConverter.fromUuid(uuid) as CharSequence })
```

Reply:
This class is no longer required :D